hedyorg/hedy
# [FIX] Stop trimming code on change in embedded editor

**Description**

Changes the call to the function `get_active_and_trimmed_code` in the change listener of the editor to a new function called `getEditorContents`. The first actually changes the editor contents, trimming the code on every change. 

**Fixes #4852**

**How to test**

* Go to http://0.0.0.0:8080/embedded/1/?lang=nl&run=true&readOnly=false&program=cHJpbnQgaGFsbG8gd2VyZWxkIQ==

And check that you can add spaces to the end of a line
